### PR TITLE
Initialize basic code structure for shardingsphere-cluster

### PR DESCRIPTION
**Pull request by menghaoranss**
For #5666 .

Changes proposed in this pull request:
- revise sub modules name 
- add cluster configuration and swapper

### CODE REVIEW

File: shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/main/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapper.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.apache.shardingsphere.cluster.configuration.config.HeartBeatConfiguration;
+import org.apache.shardingsphere.cluster.configuration.yaml.YamlClusterConfiguration;
+import org.apache.shardingsphere.cluster.configuration.yaml.YamlHeartBeatConfiguration;
+import org.apache.shardingsphere.infra.yaml.swapper.YamlSwapper;
+
+/**
+ * Cluster configuration YAML swapper.
+ */
+public final class ClusterConfigurationYamlSwapper implements YamlSwapper<YamlClusterConfiguration, ClusterConfiguration> {
+    @Override
```
Comment (dongzl) [inline shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/main/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapper.java:30]: There should be a blank line here.

Comment (menghaoranss) [reply shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/main/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapper.java:30]: thanks, i will fix it.

File: shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/test/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapperTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import org.apache.shardingsphere.cluster.configuration.yaml.YamlHeartBeatConfiguration;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThat;
+
+public final class ClusterConfigurationYamlSwapperTest {
+    
+    private static final String SQL = "select 1;";
```
Comment (kimmking) [inline shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/test/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapperTest.java:32]: remove ;

Comment (menghaoranss) [reply shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-configuration/src/test/java/org/apache/shardingsphere/cluster/configuration/swapper/ClusterConfigurationYamlSwapperTest.java]: removed.

**Pull request merged**